replica/database.cc

```diff
@@ -2577,6 +2577,10 @@ future<> database::truncate_table_on_all_shards(sharded<database>& sharded_db, s
     if (!sharded_db.local().get_config().auto_snapshot()) {
         with_snapshot = false;
     }
+    if (with_snapshot && !table_shards->get_storage_options().is_local_type()) {
+        dblog.warn("Skipping snapshot for {}.{} because it's not of local type", s->ks_name(), s->cf_name());
```

Comment:
I think the message could be clearer, as a user I wouldn't understand what "a snapshot" refers to, and what is a "local type".
Maybe something like `Not snapshotting deleted table {}.{} despite auto_snapshot=true - table is not using local disk.`.